src/documents/firstDiscoveryServer.md

```diff
@@ -0,0 +1,365 @@
+---
+title: First Discovery Server
+layout: default
+category: Overview
+---
+
+## Launching ##
+
+The First Discovery Server can be launched as a [Kettle](https://github.com/amb26/kettle/blob/KETTLE-32/README.md) application by making use of [Kettle Configs](https://github.com/amb26/kettle/blob/KETTLE-32/README.md#structure-of-a-kettle-config).
```

Comment:
It is not a Kettle server, it is just launch able with Kettle configs. Which i guess could actually be used to launch any grade really.